app/services/chatbot/chatbot-common.ts

```diff
@@ -330,7 +330,7 @@ export class ChatbotCommonService extends PersistentStatefulService<IChatbotComm
   openGamblePreferencesWindow() {
     this.windowsService.showWindow({
       componentName: 'ChatbotGamblePreferencesWindow',
-      title: $t('Chatbot Gamble Preferences'),
+      title: $t('Cloudbot Gamble Preferences'),
```

Comment:
We are in the process of translation now. Please update translation strings in `en-US` folder

Reply:
Done.